testing-examples/naming-runs/naming-test-runs.md

```diff
@@ -0,0 +1,37 @@
+# Want to name your test runs? Here's how!
+You can easily name langsmith test runs by setting the `project_name` parameter:
+
+```python
+def run_on_dataset(
```

Comment:
if this changes in source code, it may be confusing... maybe better to not include this?